ospfd/ospf_ext.c

```diff
@@ -602,8 +639,8 @@ static void ospf_ext_link_ism_change(struct ospf_interface *oi, int old_status)
 		exti->instance = get_ext_link_instance_value();
 		exti->type = OPAQUE_TYPE_EXTENDED_LINK_LSA;
 
-		zlog_debug("EXT (%s): Set %s SID to interface %s ", __func__,
-			   exti->stype == ADJ_SID ? "Adj." : "LAN Adj.",
+		zlog_debug("EXT (%s): Set %sSID to interface %s ", __func__,
```

Comment:
open debug - should be under IS_DEBUG_OSPF_SR?

Reply:
Same as above